### DISCUSSION TITLE
Sender name

### DISCUSSION THREAD

**Discussion by mattiamarras** [chatwoot/chatwoot#4308] [Q&A] [answered]
Hi, i would like to hide the agent name in the sender of the emails, how can I do it?
I don't want that that customer see, for example, John from Corporate but only the name of the brand for all the agents.

Thanks.

M.

**Comment by sojan-official** [comment, marked as answer]
At the moment it is not possible. added https://github.com/chatwoot/chatwoot/issues/4315 to track the same.
